pkg/local_workflows/code_workflow.go

```diff
@@ -36,11 +39,40 @@ func GetCodeFlagSet() *pflag.FlagSet {
 // WORKFLOWID_CODE defines a new workflow identifier
 var WORKFLOWID_CODE workflow.Identifier = workflow.NewWorkflowIdentifier(codeWorkflowName)
 
+func getSastEnabled(engine workflow.Engine) configuration.DefaultValueFunction {
+	callback := func(existingValue interface{}) interface{} {
+		if existingValue == nil {
```

Comment:
**nit (non-blocking):**  Consider exiting early when existingValue != nil rather than using the if/else pattern.